sql/core/src/test/scala/org/apache/spark/sql/execution/command/AlterTableDropPartitionSuiteBase.scala

```diff
@@ -181,4 +181,38 @@ trait AlterTableDropPartitionSuiteBase extends QueryTest with DDLCommandTestUtil
       checkPartitions(t)
     }
   }
+
+  test("SPARK-34138: keep dependents cashed after table altering") {
```

Comment:
`cashed` -> `cached`.  😉

Reply:
IMHO, it has slightly different meaning. @srowen WDYT, how to say "table dependents are still cached after table altering". Maybe just say exactly like this?